### ISSUE TITLE
Duel/thorn effect bug

### ISSUE THREAD

**Issue by KenDonnut** [Wynncraft/Issues#5623]
When I am dueling and my opponent has low HP and he uses a high damage spell on me, I win the duel and die (I lose soul points and get teleported to the nearest city). I assume that the thorns damage gets applied first, after which he is defeated, I win the duel and only after that is the actual damage applied to me, which actually kills me.

P.s. My thorns stats is 82%

**Comment by Moe-Ronickah** [comment]
Duplicate of #3559 .  Please follow this issue, there.
